contrib/kind.sh

```diff
@@ -205,307 +203,339 @@ print_params()
      echo ""
 }
 
-# In order to allow providing arguments with spaces, e.g. "-vconsole:info -vfile:info" 
-# the original command <parse_args $*> was replaced by <parse_args "$@">
-parse_args "$@"
-
-# ensure j2 renderer installed
-pip install wheel --user
-pip freeze | grep j2cli || pip install j2cli[yaml] --user
-export PATH=~/.local/bin:$PATH
-
-# Set default values
-KIND_CLUSTER_NAME=${KIND_CLUSTER_NAME:-ovn}
-K8S_VERSION=${K8S_VERSION:-v1.20.0}
-OVN_GATEWAY_MODE=${OVN_GATEWAY_MODE:-local}
-KIND_INSTALL_INGRESS=${KIND_INSTALL_INGRESS:-false}
-OVN_HA=${OVN_HA:-false}
-KIND_CONFIG=${KIND_CONFIG:-./kind.yaml.j2}
-KIND_REMOVE_TAINT=${KIND_REMOVE_TAINT:-true}
-KIND_IPV4_SUPPORT=${KIND_IPV4_SUPPORT:-true}
-KIND_IPV6_SUPPORT=${KIND_IPV6_SUPPORT:-false}
-OVN_HYBRID_OVERLAY_ENABLE=${OVN_HYBRID_OVERLAY_ENABLE:-false}
-OVN_DISABLE_SNAT_MULTIPLE_GWS=${OVN_DISABLE_SNAT_MULTIPLE_GWS:-false}
-OVN_EMPTY_LB_EVENTS=${OVN_EMPTY_LB_EVENTS:-false}
-OVN_MULTICAST_ENABLE=${OVN_MULTICAST_ENABLE:-false}
-KIND_ALLOW_SYSTEM_WRITES=${KIND_ALLOW_SYSTEM_WRITES:-false}
-OVN_IMAGE=${OVN_IMAGE:-local}
-MASTER_LOG_LEVEL=${MASTER_LOG_LEVEL:-5}
-NODE_LOG_LEVEL=${NODE_LOG_LEVEL:-5}
-DBCHECKER_LOG_LEVEL=${DBCHECKER_LOG_LEVEL:-5}
-OVN_LOG_LEVEL_NORTHD=${OVN_LOG_LEVEL_NORTHD:-"-vconsole:info -vfile:info"}
-OVN_LOG_LEVEL_NB=${OVN_LOG_LEVEL_NB:-"-vconsole:info -vfile:info"}
-OVN_LOG_LEVEL_SB=${OVN_LOG_LEVEL_SB:-"-vconsole:info -vfile:info"}
-OVN_LOG_LEVEL_CONTROLLER=${OVN_LOG_LEVEL_CONTROLLER:-"-vconsole:info"}
-OVN_LOG_LEVEL_NBCTLD=${OVN_LOG_LEVEL_NBCTLD:-"-vconsole:info"}
-
-# Input not currently validated. Modify outside script at your own risk.
-# These are the same values defaulted to in KIND code (kind/default.go).
-# NOTE: KIND NET_CIDR_IPV6 default use a /64 but OVN have a /64 per host
-# so it needs to use a larger subnet
-#  Upstream - NET_CIDR_IPV6=fd00:10:244::/64 SVC_CIDR_IPV6=fd00:10:96::/112
-NET_CIDR_IPV4=${NET_CIDR_IPV4:-10.244.0.0/16}
-SVC_CIDR_IPV4=${SVC_CIDR_IPV4:-10.96.0.0/16}
-NET_CIDR_IPV6=${NET_CIDR_IPV6:-fd00:10:244::/48}
-SVC_CIDR_IPV6=${SVC_CIDR_IPV6:-fd00:10:96::/112}
-JOIN_SUBNET_IPV4=${JOIN_SUBNET_IPV4:-100.64.0.0/16}
-JOIN_SUBNET_IPV6=${JOIN_SUBNET_IPV6:-fd98::/64}
-KIND_NUM_MASTER=1
-if [ "$OVN_HA" == true ]; then
-  KIND_NUM_MASTER=3
-  KIND_NUM_WORKER=${KIND_NUM_WORKER:-0}
-else
-  KIND_NUM_WORKER=${KIND_NUM_WORKER:-2}
-fi
-
-print_params
+install_j2_renderer() {
+  # ensure j2 renderer installed
+  pip install wheel --user
+  pip freeze | grep j2cli || pip install j2cli[yaml] --user
+  export PATH=~/.local/bin:$PATH
+}
 
-set -euxo pipefail
+set_params() {
```

Comment:
`set_default_params`  or `defaulting` , something that matchs better the function

Reply:
sure, changing to `set_default_params`